src/Symfony/Component/Cache/composer.json

```diff
@@ -27,7 +27,8 @@
         "symfony/cache-contracts": "^2.5|^3",
         "symfony/deprecation-contracts": "^2.5|^3.0",
         "symfony/service-contracts": "^2.5|^3",
-        "symfony/var-exporter": "^6.4|^7.0"
+        "symfony/var-exporter": "^6.4|^7.0",
+        "psr/clock": "^1.0"
```

Comment:
this line is not needed, it should be removed

Reply:
done